mibidata/mibi_image.py

```diff
@@ -0,0 +1,462 @@
+"""Base module for working with MIBI images.
+
+Copyright (C) 2018 Ionpath, Inc.  All rights reserved."""
+
+import datetime
+import os
```

Comment:
Unused import

Reply:
Nevermind, it's used in a format string and my editor hasn't figured that out yet.